src/pages/index.js

```diff
@@ -48,15 +108,13 @@ const enableValidation = (selectors) => {
   });
 };
 
-cardSection.renderItems(initialCards);
+// cardSection.renderItems(initialCards);
 enableValidation(selectors);
 profileEdit.setEventListeners();
 addCardModal.setEventListeners();
 imageModal.setEventListeners();
-
-function updateUserInfo({ name, description }) {
-  currentUserInfo.setUserInfo({ name, description });
-}
+avatarModalPopup.setEventListeners();
+deleteModalPopup.setEventListeners();
 
 function setFormInfo(nameSelector, detailsSelector) {
   const formName = document.querySelector(nameSelector);
```

Comment:
**Could be improved**

It’s better to find **constants** only 1 time at the top of the file not to waste resources on searching them again and again when you call a method (function) because searching is a very **hard** operation for the browser engine